tests/unit/hca/test_submission.py

```diff
@@ -7,73 +7,113 @@
         # Then
-        self.assertTrue(self.submission.contains_entity(self.test_case))
-        self.assertTrue(self.submission.contains_entity_by_uuid(self.test_type, self.test_uuid))
-        self.assertTrue(self.submission.get_entity(self.test_type, self.test_id))
+        self.assertTrue(self.submission.contains_entity(self.project_json))
+        self.assertTrue(self.submission.contains_entity_by_uuid(self.entity_type_1, self.test_uuid1))
+        self.assertTrue(self.submission.get_entity(self.entity_type_1, self.test_id1))
 
     def test_entity_attributes_are_equal_to_source(self):
         # Then
-        self.assertEqual(self.test_case, self.test_entity.attributes)
+        self.assertEqual(self.project_json, self.test_project.attributes)
 
     def test_added_entity_is_equal_to_entity_returned_from_submission(self):
         # When
-        returned_entity = self.submission.get_entity(self.test_type, self.test_id)
+        returned_entity = self.submission.get_entity(self.entity_type_1, self.test_id1)
         # Then
-        self.assertEqual(self.test_entity, returned_entity)
+        self.assertEqual(self.test_project, returned_entity)
 
     def test_added_entity_is_equal_to_entity_returned_from_hca_submission_by_uuid(self):
         # When
-        returned_entity = self.submission.get_entity_by_uuid(self.test_type, self.test_uuid)
+        returned_entity = self.submission.get_entity_by_uuid(self.entity_type_1, self.test_uuid1)
         # Then
-        self.assertEqual(self.test_entity, returned_entity)
+        self.assertEqual(self.test_project, returned_entity)
 
     def test_adding_a_duplicate_entity_returns_the_original_entity(self):
         # When
-        returned_entity = self.submission.map_ingest_entity(self.test_case)
+        returned_entity = self.submission.map_ingest_entity(self.project_json)
         # Then
-        self.assertEqual(self.test_entity, returned_entity)
+        self.assertEqual(self.test_project, returned_entity)
 
     def test_duplication_entity_with_differing_content_is_ignored(self):
         # Given
         ignored_uuid = random_uuid()
-        duplicate_test_case = make_ingest_entity(self.test_type, self.test_id, ignored_uuid)
+        duplicate_test_case = make_ingest_entity(self.entity_type_1, self.project_schema_type, self.test_id1, ignored_uuid)
         # When
         duplicate_entity = self.submission.map_ingest_entity(duplicate_test_case)
         # Then
-        self.assertEqual(self.test_entity, duplicate_entity)
-        self.assertEqual(self.test_uuid, self.test_entity.attributes['uuid']['uuid'])
-        self.assertNotEqual(ignored_uuid, self.test_entity.attributes['uuid']['uuid'])
+        self.assertEqual(self.test_project, duplicate_entity)
+        self.assertEqual(self.test_uuid1, self.test_project.attributes['uuid']['uuid'])
+        self.assertNotEqual(ignored_uuid, self.test_project.attributes['uuid']['uuid'])
```

Comment:
Should an exception be called for this case? If so, can you test that the exception is raised?